api/__tests__/cache/projects.test.js

```diff
@@ -311,7 +311,7 @@ describe('Test the projects/resources cache', () => {
       name: fakeJiraItApiResults.data.issues[randomItemIndex].fields.summary,
       description: fakeJiraItApiResults.data.issues[randomItemIndex].fields.description,
       client: fakeJiraItApiResults.data.issues[randomItemIndex].fields.customfield_10027,
-      video: fakeJiraItApiResults.data.issues[randomItemIndex].fields.customfield_10159,
+      video_webpage: fakeJiraItApiResults.data.issues[randomItemIndex].fields.customfield_10159,
```

Comment:
Would there need to be any verification that there is a video_webPage for the project or since there will be the empty field in the Airtable regardless should it be hardcoded similar to the required field?